features/users.feature

```diff
@@ -16,53 +16,28 @@ Feature: Users
     Then I should see "Security Notification"
     And I should see "This is a U.S. General Services Administration Federal Government computer system"
 
-  # to be updated in SRCH-862 for login.gov
+  # to be updated in SRCH-1237 for login.gov
   @wip
```

Comment:
This will remain commented out till we redirect the user to the edit page. I did cleanup some of it that wont be used.

Reply:
👍